projects/opal-frontend-common/components/custom/custom-account-information/custom-account-information.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'opal-lib-custom-account-information , [opal-lib-custom-account-information]',
```

Comment:
Theres a space before the comma

```suggestion
  selector: 'opal-lib-custom-account-information, [opal-lib-custom-account-information]',
```